apps/condo/pages/_app.tsx

```diff
@@ -62,59 +63,78 @@ const ANT_LOCALES = {
     en: enUS,
 }
 
-interface IMenuItemData {
-    path: string,
-    icon: React.FC,
-    label: string,
-}
-
-const ANT_DEFAULT_LOCALE = enUS
-
-const MenuItems: React.FC = () => {
-    const { link } = useOrganization()
-    const { isExpired } = useServiceSubscriptionContext()
-    const hasSubscriptionFeature = hasFeature('subscription')
-    const disabled = !link || (hasSubscriptionFeature && isExpired)
-    const { isCollapsed } = useLayoutContext()
-    const { wrapElementIntoNoOrganizationToolTip } = useNoOrganizationToolTip()
-
-    const menuItemsData: IMenuItemData[] = [{
+const itemsConfigs = {
+    canGetExternalReportIframeUrl: {
         path: 'reports',
         icon: BarChartIconNew,
         label: 'menu.Analytics',
```

Comment:
OMG O_O
I meant that you may add a new field with access to each menu item. What if we decide to use two or more "rules"?
```suggestion
    {
        path: 'reports',
        icon: BarChartIconNew,
        label: 'menu.Analytics',
        access: ['canGetExternalReportIframeUrl'],
```